crates/bevy_color/src/linear_rgba.rs

```diff
@@ -49,6 +65,18 @@ impl LinearRgba {
         }
     }
 
+    /// Construct a new [`LinearRgba`] color with the same value for all channels and an alpha of 1.0.
+    ///
+    /// A value of 0.0 is black, and a value of 1.0 is white.
+    pub const fn grey(value: f32) -> Self {
```

Comment:
The spelling "grey" is use more frequently in British English, whereas American English tends to prefer "gray". However, usage is not consistent.